tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunner.java

```diff
@@ -77,6 +78,13 @@ public void generateReports(ITestContext context) throws IOException {
         generateReport();
     }
 
+    private static HashSet<String> getSkippedTestLabels() {
+        HashSet<String> hashSet = new HashSet<>();
```

Comment:
Try to follow programming to an interface when ever possible
```suggestion
    private static Set<String> getSkippedTestLabels() {
        Set<String> hashSet = new HashSet<>();
```